src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -95,17 +95,27 @@ class DepositServlet(app: EasyDepositApiApp)
       } yield Ok(body = s"""{"doi":"$doi"}""", headers = Map(contentTypeJson))
     }.getOrRecoverWithActionResult
   }
+
+  private def fixLicenseUrl(maybeSchemedValue: Option[SchemedValue]) = {
+    def replace(str: String) = {
+      str.replaceAll(".ezproxy[A-Za-z0-9.]+/", "/")
```

Comment:
`.` in a regex refers to "any char". In this case we want a literal dot. I'll merge it and correct it afterwards, though.